src/main/scala/io/confluent/examples/streams/algebird/CMSStore.scala

```diff
@@ -194,7 +194,7 @@ class CMSStore[T: CMSHasher](override val name: String,
       name,
       Serdes.Integer(),
       TopCMSSerde[T])
-    changeLogger = new CMSStoreChangeLogger[Integer, TopCMS[T]](name, context, serdes)
+    changeLogger = new CMSStoreChangeLogger[Integer, TopCMS[T]](name, context, serdes, name)
```

Comment:
Are you sure it is fine to use the same name for the state store and the processor node ID?

Reply:
I actually set the processor-name so it should match up now.